package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qpp-measures-data",
-  "version": "1.8.9",
+  "version": "1.9.0",
```

Comment:
Because this is a bugfix let's just bump the patch version: `1.8.10`

Reply:
Got it. Will change.

Reply:
good call!